src/components/Button/Button.react.js

```diff
@@ -59,19 +47,8 @@ const Button = ({
       btn: true,
       [`btn-${size}`]: !!size,
       [`btn-block`]: block,
-      [`btn-primary`]: primary && !outline,
-      [`btn-secondary`]: secondary && !outline,
-      [`btn-success`]: success && !outline,
-      [`btn-info`]: info && !outline,
-      [`btn-warning`]: warning && !outline,
-      [`btn-danger`]: danger && !outline,
+      [`btn-outline-${color}`]: outline && !!color,
       [`btn-link`]: link,
-      [`btn-outline-primary`]: outline && primary,
-      [`btn-outline-secondary`]: outline && secondary,
-      [`btn-outline-success`]: outline && success,
-      [`btn-outline-info`]: outline && info,
-      [`btn-outline-warning`]: outline && warning,
-      [`btn-outline-danger`]: outline && danger,
       disabled: disabled,
       [`btn-${color}`]: !!color,
```

Comment:
Although it doesn't cause any issues, to try and stay in line with the Tabler docs, outline buttons should only have `btn-outline-color` rather than both `btn-color` and `btn-color-outline`